tox.ini

```diff
@@ -8,6 +8,7 @@ passenv = *
 extras = all
 deps =
   poetry==1.3.2
+  charset-normalizer==3.1.0
```

Comment:
Added because poetry was installing version 3.2.0 and then uninstalling to install 3.1.0 (via `tox -e test-unit`), which was causing a file permissions error in Windows for some reason.